_locales/pl/messages.json

```
@@ -0,0 +1,387 @@
+{
+	"extension_description": {
+		"message": "Dodaje linki SteamDB i nowe funkcjonalności w sklepie i społeczności Steam. Wyświetla najniższe ceny gier i ich statystyki."
+	},
+
+	"steamdb_options": {
+		"message": "Opcje SteamDB"
+	},
+	"steamdb_calculator": {
+		"message": "Kalkulator SteamDB"
+	},
+	"steamstatus": {
+		"message": "Sprawdź steamstat.us"
+	},
+	"steamstatus_downtime": {
+		"message": "Wygląda na to, że Steam doświadcza pewnych awarii. "

```

```suggestion
		"message": "Wygląda na to, że występują problemy z połączeniem ze Steam. "

```

Obecnie brzmi to bardzo nienaturalnie.